samples/k8s-batch-examples.md

```diff
@@ -15,675 +13,698 @@ os.environ['MESH_IP'] = MESH_IP
-```
-
-    {
-      "model_name": "",
-      "id": "1cee209f-be47-401b-b7e7-5ce5e5c78b40",
-      "parameters": {
-        "batch_index": 1
-      },
-      "outputs": [
-        {
-          "name": "OUTPUT0",
-          "shape": [
-            1,
-            16
-          ],
-          "datatype": "INT32",
-          "data": [
-            115,
-            69,
-            97,
-            112,
-            73,
-            106,
-            58,
-            182,
-            114,
-            66,
-            64,
-            110,
-            100,
-            24,
-            22,
-            77
-          ]
-        },
-        {
-          "name": "OUTPUT1",
-          "shape": [
-            1,
-            16
-          ],
-          "datatype": "INT32",
-          "data": [
-            -77,
-            33,
-            25,
-            -52,
-            -49,
-            -88,
-            -48,
-            0,
-            -50,
-            26,
-            -44,
-            46,
-            -2,
-            18,
-            -6,
-            -47
-          ]
-        }
-      ]
-    }
+```json
+{
+  "model_name": "",
+  "id": "1cee209f-be47-401b-b7e7-5ce5e5c78b40",
+  "parameters": {
+    "batch_index": 1
+  },
+  "outputs": [
+    {
+      "name": "OUTPUT0",
+      "shape": [
+        1,
+        16
+      ],
+      "datatype": "INT32",
+      "data": [
+        115,
+        69,
+        97,
+        112,
+        73,
+        106,
+        58,
+        182,
+        114,
+        66,
+        64,
+        110,
+        100,
+        24,
+        22,
+        77
+      ]
+    },
+    {
+      "name": "OUTPUT1",
+      "shape": [
+        1,
+        16
+      ],
+      "datatype": "INT32",
+      "data": [
+        -77,
+        33,
+        25,
+        -52,
+        -49,
+        -88,
+        -48,
+        0,
+        -50,
+        26,
+        -44,
+        46,
+        -2,
+        18,
+        -6,
+        -47
+      ]
+    }
+  ]
+}
 
+```
 
 ## Cleanup
 
-
-```python
-!kubectl delete -f models/sklearn-iris-gs.yaml -n ${NAMESPACE}
-!kubectl delete -f pipelines/iris.yaml -n ${NAMESPACE}
+```bash
+kubectl delete -f models/sklearn-iris-gs.yaml -n ${NAMESPACE}
+kubectl delete -f pipelines/iris.yaml -n ${NAMESPACE}
 ```
 
-    model.mlops.seldon.io "iris" deleted
-    pipeline.mlops.seldon.io "iris-pipeline" deleted
-
-
+```
+model.mlops.seldon.io "iris" deleted
+pipeline.mlops.seldon.io "iris-pipeline" deleted
 
-```python
-!kubectl delete -f models/tfsimple1.yaml -n ${NAMESPACE}
-!kubectl delete -f pipelines/tfsimple.yaml -n ${NAMESPACE}
 ```
 
-    model.mlops.seldon.io "tfsimple1" deleted
-    pipeline.mlops.seldon.io "tfsimple" deleted
+```bash
+kubectl delete -f models/tfsimple1.yaml -n ${NAMESPACE}
+kubectl delete -f pipelines/tfsimple.yaml -n ${NAMESPACE}
+```
 
+```
+model.mlops.seldon.io "tfsimple1" deleted
+pipeline.mlops.seldon.io "tfsimple" deleted
 
+```
 
 ```python
```

Comment:
:information_source: There are at least a few files with empty cells at the end.  Another one for a follow-up PR would be to identify and remove these redundant cells as they're purely noise for viewers of the Markdown files.